web-common/tests/utils/start-rill-dev.ts

```diff
@@ -0,0 +1,109 @@
+import { asyncWaitUntil } from "@rilldata/web-common/lib/waitUtils";
+import { getOpenPort } from "@rilldata/web-common/tests/utils/get-open-port";
+import { spawnAndMatch } from "@rilldata/web-common/tests/utils/spawn";
+import axios from "axios";
+import { spawn } from "node:child_process";
+import { cpSync, existsSync, mkdirSync, rmSync } from "node:fs";
+import { join } from "node:path";
+import type { Page } from "playwright";
+import treeKill from "tree-kill";
+
+export const TestTempDirectory = "playwright";
+
+export async function startRillDev(
+  page: Page,
+  use: (page: Page) => Promise<void>,
+  {
+    cliHome,
+    switchEnv,
+    projectDir,
+  }: {
+    cliHome?: string;
+    switchEnv?: boolean;
```

Comment:
Can we remove this? I'd expect us always to be in the `dev` context during these tests.